api/src/main/java/org/apache/cloudstack/consoleproxy/ConsoleAccessManager.java

```diff
@@ -18,9 +18,31 @@
 
 import com.cloud.utils.component.Manager;
 import org.apache.cloudstack.api.command.user.consoleproxy.ConsoleEndpoint;
+import org.apache.cloudstack.framework.config.ConfigKey;
 
 public interface ConsoleAccessManager extends Manager {
 
+    ConfigKey<Boolean> ConsoleSessionCleanupEnabled = new ConfigKey<>("Advanced", Boolean.class,
+            "console.session.cleanup.enabled",
+            "true",
+            "Enables/disables the console session cleanup thread.",
+            true,
```

Comment:
If we change the values, we need to restart cloudstack management server, right ?
if yes, these configurations are not dynamic.

Reply:
Yes, my mistake, thanks